registry_resolver/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-mod registry_client;
+pub mod registry_client;
```

Comment:
This shouldn't be public

Reply:
this is part of trait declaration on server.rs here, not sure it can be avoided https://github.com/knox-networks/core/pull/693/files#diff-4c7161a5afc4e79f458609693cf8b7b4dd07db81b67884e7c444ba626253b1feR453